invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/Results.js

```diff
@@ -28,7 +28,7 @@ export const Results = ({ currentResultsState = {} }) => {
     useContext(SearchConfigurationContext);
   const multipleLayouts = layoutOptions.listView && layoutOptions.gridView;
   return (
-    (total) && (
+    (total || null) && (
```

Comment:
```suggestion
    total === 0 && (
```

Reply:
unfortunately still displaying zero if implemented this way